client/webpack.config.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { DefinePlugin } = require('webpack');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/main.tsx',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true, // 빌드 시 dist 폴더 정리
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './index.html', // 템플릿 HTML
+      filename: 'index.html', // 출력될 HTML 파일 이름
+      inject: true, // <script> 태그 자동 삽입
+    }),
+    new DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify('development'),
+      VERSION: JSON.stringify('1.0.0'),
+      __DEV__: JSON.stringify(true),
+    }),
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.(ts|tsx)$/, // .ts와 .tsx 파일을 대상으로
+        use: [
+          {
+            loader: 'babel-loader',
+          },
+        ],
```

Comment:
`loader` 키를 명시해주신 이유가 옵션을 추가하거나 확장하기 위함인가요?? 통일감을 위해 다음과 같이 맞춰볼 수도 있을 것 같아서요!

익셉의 생각이 궁금합니다~!
```suggestion
        use: ['babel-loader'],
```

Reply:
아뇨 확장하기 위함 아니었습니다! 저도 통일감있게 하는 의견 동의합니다 ㅎㅎ!
c736f92f0526d029100ccb7ca735c64612af6b02